src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CommitManager.cs

```diff
@@ -124,7 +124,7 @@ internal CommitManager(ITextView textView, RecentItemsManager recentItemsManager
                 return new AsyncCompletionData.CommitResult(isHandled: true, AsyncCompletionData.CommitBehavior.None);
             }
 
-            if (!session.Properties.TryGetProperty(CompletionSource.TriggerSnapshot, out ITextSnapshot triggerSnapshot))
+            if (!session.Properties.TryGetProperty(CompletionSource.TriggerLocation, out SnapshotPoint triggerLocation))
             {
                 // Need the trigger snapshot to calculate the span when the commit changes to be applied.
```

Comment:
>snapshot [](start = 36, length = 8)

correct this comment. But better please consider getting the object not from the property bag but form a session property.

Reply:
Fixed